src/react/src/components/Filters/SectorFilter.jsx

```diff
@@ -20,33 +27,61 @@ function SectorFilter({
     fetchingOptions,
     hideSectorData,
     isSideBarSearch,
+    embed,
+    groupedSectorOptions,
+    fetchingGroupedSectors,
+    fetchGroupedSectors,
 }) {
     if (hideSectorData) {
         return null;
     }
+
     return (
-        <div className="form__field">
-            <StyledSelect
-                name={SECTORS}
-                label="Sector"
-                options={sectorOptions || []}
-                value={sectors}
-                onChange={updateSector}
-                onFocus={() =>
-                    !sectorOptions && !fetchingSectors && fetchSectors()
-                }
-                noOptionsMessage={() =>
-                    fetchingSectors ? 'Loading..' : 'No options'
-                }
-                disabled={fetchingOptions || fetchingSectors}
-                isSideBarSearch={isSideBarSearch}
-            />
-        </div>
+        <>
```

Comment:
Do we need to wrap this with a fragment? It seems that <div className="form__field"> already acts as a wrapper.